lib/email_tracking_retriever.py

```diff
@@ -285,10 +284,20 @@ def clean_email_content(email_str) -> str:
   return email_str
 
 
-def log_in_if_necessary(driver: WebDriver):
+def log_in_if_necessary(driver: WebDriver, config):
   driver.get('https://www.amazon.com/gp/your-account/order-history/ref=ppx_yo_dt_b_orders')
   orders_containers = driver.find_elements_by_id('ordersContainer')
   if len(orders_containers) == 0:
```

Comment:
@gbrodman would you be ok with removing both these lines? Every time Chromium opens it doesn't seem to save any session (cookies/storage), unless others have that configured different. Keeping this now seems to take 5-10 seconds or so to start filling out your email.

Reply:
Yeah that works for me. I was under the impression that others were seeing the state persist between runs, but that might have been a miscommunication. I haven't seen it persist state.

Reply:
👍 Updated the PR, this should now be ready